gatsby-node.js

```diff
@@ -433,7 +409,7 @@ exports.onCreateNode = ({ node, getNode, actions }) => {
         })
       }
     }
-
+    // gatsby 5 doesn't like the `matter` it's getting from gray-matter here: "expected input to be a string or buffer"
     if (sourceInstanceName === 'changelogs') {
       const content = matter(node.internal.content, { excerpt: true, excerpt_separator: '<!-- excerpt -->' } );
```

Comment:
@jmolivas do you think this is something we should try to make work, or should we try to switch to https://www.gatsbyjs.com/plugins/gatsby-plugin-excerpts/ instead? (I'm just playing around with this to see if it'll work easily enough)